src/modules/addtask.js

```diff
@@ -0,0 +1,13 @@
+export class Task {
+  constructor(description, completed, index) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+}
+export const inputTask = document.querySelector('.input-task');
+
+export const addTask = (tasks, input) => {
+  const newTask = new Task(input.value, false, tasks.length);
+  tasks.push(newTask);
+};
```

Comment:
- [ ] Kindly add a function for editing task descriptions as stated in the [project's requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md).